localstack/services/apigateway/patches.py

```diff
@@ -432,7 +432,7 @@ def apigateway_models_RestAPI_to_dict(self):
 
     def individual_deployment(self, request, full_url, headers, *args, **kwargs):
         result = individual_deployment_orig(self, request, full_url, headers, *args, **kwargs)
```

Comment:
Because the `individual_deployment_orig` method already checks the boundaries we can remove from here.